### ISSUE TITLE
After being added in UI, incident Type & Priority did not show up  upon creation of ticket

### ISSUE THREAD

**Issue by olam-slack** [Netflix/dispatch#1927]
Not using sample data (database dropped), added incident type & priority manually but they did not show up upon creation of ticket.
The following may have shown in server log
psycopg2.errors.UndefinedTable: table "document_terms" does not exist

![image](https://user-images.githubusercontent.com/96050653/151072836-5e019922-760e-4f32-8324-4e429101ce6b.png)
![image](https://user-images.githubusercontent.com/96050653/151072848-7abae040-ec15-4a65-a6ff-5ec4fdb1d882.png)
![image](https://user-images.githubusercontent.com/96050653/151072855-165cae73-bfba-4a91-bd7a-38fe7004ebe8.png)

**Comment by mvilanova** [comment]
Did you run `dispatch database init` after dropping the database?

**Comment by olam-slack** [comment]
Yes I did.

**Comment by mvilanova** [comment]
Ok, I'll see if I can repro.

**Comment by mvilanova** [comment]
I wasn't able to reproduce. I created a new incident type and priority and they showed up in the new incident creation view.

**Comment by olam-slack** [comment]
Okay, I am not sure if the difference is that I am not using sample data (database dropped then database inited), were you doing the same?

**Comment by mvilanova** [comment]
Yes, I ran `dispatch database drop` and `dispatch database init` before testing.

**Comment by olam-slack** [comment]
Hi Marc, I am closing the ticket, I restarted everything and got it running.
